mitiq/observable/pauli.py

```diff
@@ -282,8 +282,27 @@ def _measure_in(
 
         basis_rotations = set()
         support = set()
+
+        # Find any existing measurement gates in the circuit
+        existing_measurements = []
+        measurement_tuples = list(
+            circuit.findall_operations_with_gate_type(cirq.MeasurementGate)
+        )
+        if measurement_tuples:
+            existing_measurements = [
+                measurement_tuple[1].qubits[0]
+                for measurement_tuple in measurement_tuples
+            ]
+
         for pauli in paulis.elements:
             basis_rotations.update(pauli._basis_rotations())
+            for qubit in pauli._qubits_to_measure():
+                if qubit in existing_measurements:
+                    raise ValueError(
+                        f"""More than one measaurement found for qubit: """
+                        f"""{qubit}. Only a single measurement is allowed """
+                        f"""per qubit."""
```

Comment:
```suggestion
                        f"More than one measaurement found for qubit: "
                        "{qubit}. Only a single measurement is allowed "
                        "per qubit."
```

AFAIK, the f-string only need to be stated once and then it will apply to the whole string. (suggestion is modulo formatting)

Reply:
Turns out Ruff gets mad with just the single f-string stated since the first line does not have any placeholders.